resources/META-INF/plugin.xml

```diff
@@ -300,6 +300,7 @@
   <depends optional="true" config-file="idea-contribs.xml">com.intellij.modules.java</depends>
 
   <!-- Contributes Android Studio-specific features and implementations. -->
+  <!--suppress PluginXmlValidity -->
```

Comment:
Do we need this in plugin.xml.template too? Asking because I don't know, but we can add if needed.

Reply:
I'm not sure.  If it's just for development maybe not?  As you say, we can add if needed.